### ISSUE TITLE
[HTML5 Runtime] Struct `[$ "property"]` accessor missing hash in generated code

### ISSUE THREAD

**Issue by thennothinghappened** [YoYoGames/GameMaker-Bugs#4391]
### Description
In the latest Beta Runtime (**Beta IDE v2024.200.0.461 Beta Runtime v2024.200.0.477**) - I haven't tested earlier yet unfortunately - the `[$ "property"]` accessor doesn't actually write the hash for that property in the resulting call to `struct_get_from_hash`.

### Steps To Reproduce
Here's a quick snippet to show the issue:
```gml
var struct = {
	test: "hi!"
};

var dot = struct.test;
var hash = struct[$ "test"];

show_message($"dot: {dot}, hash: {hash}");
```

On any other platform, you get the message *"dot: hi!, hash: hi!"*, where on HTML5 you get *"dot: hi!, hash: undefined"*.

As for the example snippet, here's the relevant generated JS for that:
```js
function gml_GlobalScript_Struct_Accessors_Test(_inst, _other)
{
  var gmlstruct = __yy_gml_object_create(
    _inst,
    (
      gml_GlobalScript_Struct_Accessors_Test.prototype.gml___struct___0 = __yy_method(_inst, gml_Script____struct___0)
    )
  );
  var gmldot = yyInst(_inst, _other, gmlstruct).gmltest;
  // Second argument is blank here.
  var gmlhash = struct_get_from_hash(gmlstruct, );
  show_message(string('dot: {0}, hash: {1}', gmldot, gmlhash));
}
function gml_Script____struct___0(_inst, _other)
{
  if (_inst.__yyIsGMLObject)
  {
    _inst.__type = 'gml_Script____struct___0'
  }
  if (gml_Script____struct___0.prototype.__type === undefined)
  {
    gml_Script____struct___0.prototype.__type = 'gml_Script____struct___0'
  }
  {
    if (_inst.__yyIsGMLObject) Object.setPrototypeOf(_inst, gml_Script____struct___0.prototype);
    {
      _inst.gmltest = 'hi!';
    }
  }
}
```

[16ab0a64-869c-455f-89cc-99f59207f97d](https://api.gamemaker.io/api/github/downloads/16ab0a64-869c-455f-89cc-99f59207f97d)

**Comment by rwkay** [comment]
Fixed in 2024.2

**Comment by mgeddesGM** [comment]
verified as of  IDE v2024.200.0.469  Runtime v2024.200.0.485
